From: Dave Young <dyoung@redhat.com>
To: kexec@lists.infradead.org
Cc: linux-efi@vger.kernel.org, x86@kernel.org, mjg59@srcf.ucam.org,
 hpa@zytor.com, James.Bottomley@HansenPartnership.com, vgoyal@redhat.com,
 ebiederm@xmission.com, horms@verge.net.au, bp@alien8.de, greg@kroah.com,
 matt@console-pimps.org, toshi.kani@hp.com
Subject: [PATCH v4 1/4] build fix: include x86-linux.h in x86-linux-setup.h
Date: Fri, 20 Dec 2013 18:05:44 +0800
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
Content-Type: text/plain; charset=utf-8

There's build warnings about using struct x86_linux_param_header *
in x86-linux-setup.h, it is declared in x86-linux.h

Fix it by include x86-linux.h in x86-linux-setup.h

Signed-off-by: Dave Young <dyoung@redhat.com>
---
 kexec/arch/i386/x86-linux-setup.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kexec/arch/i386/x86-linux-setup.h b/kexec/arch/i386/x86-linux-setup.h
index 09aed4d..2afe97c 100644
--- a/kexec/arch/i386/x86-linux-setup.h
+++ b/kexec/arch/i386/x86-linux-setup.h
@@ -1,5 +1,6 @@
 #ifndef X86_LINUX_SETUP_H
 #define X86_LINUX_SETUP_H
+#include <x86/x86-linux.h>
 
 void init_linux_parameters(struct x86_linux_param_header *real_mode);
 void setup_linux_bootloader_parameters_high(
-- 
1.8.3.1

